src/Datadog.Trace.ClrProfiler.Native/integration_loader.cpp

```diff
@@ -12,33 +12,30 @@ namespace trace
 
 using json = nlohmann::json;
 
-std::vector<Integration> LoadIntegrationsFromEnvironment()
+void LoadIntegrationsFromEnvironment(std::vector<IntegrationMethod>& integrationMethods, const bool isCallTargetEnabled,
+                                     const bool isNetstandardEnabled,
+                                     const std::vector<WSTRING> disabledIntegrationNames)
 {
-    std::vector<Integration> integrations;
-    for (const auto f : GetEnvironmentValues(environment::integrations_path))
+    for (const WSTRING filePath : GetEnvironmentValues(environment::integrations_path))
```

Comment:
you can have `const WSTRING&` here, it will avoid a copy

Reply:
done in 1aa6b9d